tools/reviewer/web/src/app/dashboard/diff/diff-header/diff-header-content/diff-header-content.component.ts

```diff
@@ -0,0 +1,143 @@
+import { Component, Input, OnInit } from '@angular/core';
+import { FormControl } from '@angular/forms';
+import { MatDialog, MatDialogRef } from '@angular/material';
+import { debounceTime, distinctUntilChanged } from 'rxjs/operators';
+
+import { Diff, Reviewer } from '@/shared/proto';
+import {
+  AuthService,
+  FirebaseService,
+  NotificationService,
+} from '@/shared/services';
+import { AddUserDialogComponent } from '../add-user-dialog';
+import { DiffHeaderService } from '../diff-header.service';
+
+@Component({
+  selector: 'diff-header-content',
+  templateUrl: './diff-header-content.component.html',
+  styleUrls: ['./diff-header-content.component.scss'],
+  providers: [DiffHeaderService],
+})
+export class DiffHeaderContentComponent implements OnInit {
+  textareaControl: FormControl = new FormControl();
+  @Input() diff: Diff;
+
+  constructor(
+    public authService: AuthService,
+    public firebaseService: FirebaseService,
+    public notificationService: NotificationService,
+    public diffHeaderService: DiffHeaderService,
+    public dialog: MatDialog,
+  ) {
+    // Save description when it's changed
```

Comment:
This is cool, but I'm not sure it's the desired behavior.
What if the user wants to undo?

Reply:
Yeah, you're right?
What is the desired behavior though?

Reply:
I think the same as other fields - click on edit -> edit -> save/cancel.